components/device-mgt/org.wso2.carbon.device.mgt.ui/src/main/resources/jaggeryapps/devicemgt/app/pages/cdmf.page.policies/policies.js

```diff
@@ -35,15 +35,18 @@ function onRequest(context) {
         var policyListToView = response["content"];
         page["policyListToView"] = policyListToView;
         var policyCount = policyListToView.length;
-        if (policyCount == 0) {
+        switch (policyCount) {
+            case 0:
             page["policyListingStatusMsg"] = "No policy is available to be displayed.";
             page["saveNewPrioritiesButtonEnabled"] = false;
             page["noPolicy"] = true;
-        } else if (policyCount == 1) {
+                break;
```

Comment:
Ack, Fixed in commit https://github.com/wso2/carbon-device-mgt/pull/304/commits/d2daa0507eb9eb1b5d149fde50e305634e0f6b51